src/datastore/pg-write-store.ts

```diff
@@ -96,7 +101,9 @@ class MicroblockGapError extends Error {
  */
 export class PgWriteStore extends PgStore {
   readonly isEventReplay: boolean;
-  private cachedParameterizedInsertStrings = new Map<string, string>();
+  protected get closeTimeout(): number {
+    return parseInt(getPgConnectionEnvValue('CLOSE_TIMEOUT', PgServer.primary) ?? '5');
```

Comment:
This is in the `PgWriteStore` class, does that mean it's only being applied to API writers? Or is this used for both writers/readers?

Reply:
There's a similar statement in `pg-store.ts`, this is only an override so I can change the PgServer to primary (it's a subclass)